src/test/suite/fixtures/recorded/languages/css/changeItem.yml

```diff
@@ -0,0 +1,29 @@
+languageId: css
+command:
+  version: 1
+  spokenForm: change item
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: collectionItem, includeSiblings: false}
+initialState:
+  documentContents: |-
+    a {
+      width: calc(100% - 80px);
```

Comment:
@AndreasArvidsson just a quick one: how do you feel about this line as an "item"?  I'd be tempted to use "field" (#43) for this one instead.  WDYT?